### PR TITLE
runcontrol_runparameters

### PR DESCRIPTION
**Pull request by jonnyh404**

### CODE REVIEW

File: ResSimpy/Nexus/DataModels/NexusSolverParameter.py [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+
+from ResSimpy.Enums.TimeSteppingMethodEnum import TimeSteppingMethod
+from ResSimpy.SolverParameter import SolverParameter
+
+
+@dataclass
+class NexusSolverParameter(SolverParameter):
+    """Data object storing the solver parameters for the Nexus Simulation runs.
+    Nexus is time-dependent for the solver parameters.
+    """
```
[the remark this answers is not in the document]
Comment (jonnyh404) [reply ResSimpy/Nexus/DataModels/NexusSolverParameter.py:13]: Prefer "|" over Optional. As per ruff UP007

**Review by kwr3 — APPROVED** [review]
Looks good.

**Review by kwr3 — APPROVED** [review]
Looks good.

**Review by cflynn3 — APPROVED** [review]
LGTM.

**Pull request merged**